governance/program/src/instruction.rs

```diff
@@ -393,11 +414,26 @@ pub enum GovernanceInstruction {
     ///       If that's required then it must be done before executing this instruction
     ///   3. `[writable]` Council Token Holding account - optional unless council is used. PDA seeds: ['governance',realm,council_mint]
     ///       The account will be created with the Realm PDA as its owner
+    ///   4. `[signer]` Payer
+    ///   5. `[]` System
+    ///   6. `[writable]` RealmConfig account. PDA seeds: ['realm-config', realm]
+    ///   7. `[]` Optional Community Voter Weight Addin Program Id    
     SetRealmConfig {
         #[allow(dead_code)]
         /// Realm config args
         config_args: RealmConfigArgs,
     },
+
+    /// Creates TokenOwnerRecord with 0 deposit amount
+    /// It's used to register TokenOwner when voter weight addin is used and the Governance program doesn't take deposits
```

Comment:
It seems like the "addin" could be better seen as a vault!  In the end, it will hold all of the tokens, correct?